mlflow/server/js/src/experiment-tracking/components/CompareRunContour.js

```diff
@@ -182,9 +171,9 @@ export class CompareRunContour extends Component {
           data={[
             // contour plot
             {
-              z,
-              x: xsSorted,
-              y: ysSorted,
+              z: zs,
+              x: xs,
+              y: ys,
```

Comment:
Does this really work?

Reply:
I attached a plot, it looks correct to me.

Reply:
Ah I see they don't need to be sorted.